examples/archive.html

```diff
@@ -108,6 +108,26 @@ <h3 id="offcanvasRightLabel">Images</h3>
                 placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
                 placeButton.innerHTML = 'Place on map';
```

Comment:
This looks super! One small spacing suggestion:
```suggestion
                placeButton.innerHTML = 'Place on map ';
```

Reply:
Ohh, alright, I'll make the necessary change

Reply:
@jywarren i have added the white space. Looks good now?